sprocs/instance_questions_next_allowed_grade.sql

```diff
@@ -0,0 +1,56 @@
+DROP FUNCTION IF EXISTS instance_questions_next_allowed_grade(bigint);
+CREATE OR REPLACE FUNCTION
+    instance_questions_next_allowed_grade (
+        IN iq_id BIGINT,
+        OUT allow_grade_date TIMESTAMPTZ,
+        OUT allow_grade_left_ms BIGINT,
+        OUT allow_grade_interval TEXT
+    )
+AS $$
+BEGIN
+    SELECT
+        MAX(gj.date + aq.grade_rate_minutes * INTERVAL '1 min')
```

Comment:
```suggestion
        MAX(gj.date + make_interval(mins => aq.grade_rate_minutes))
```

Reply:
I had to switch back, `make_interval` doesn't support double precision.